src/card/card.scss

```diff
@@ -0,0 +1,32 @@
+@import '../var-defaults.scss';
+@import '_attractions-theme.scss';
+
+.card {
+  box-shadow: $shadow-0;
+  background-color: #fff;
+  border-radius: 1.25rem;
+  padding: 1.5em;
+  overflow: hidden;
+
+  &.with-image {
```

Comment:
And the simplicity of the component also means getting rid of those styles. The only useful prop (and class, for that matter) to my mind is the one that would disable the internal padding.

Reply:
In my point of view, libraries should do as much heavy lifting and provide as much functionality as possible out-of-the-box, while not disallowing any customization. Excessive simplicity reduces the added value and increases the amount of work the user should do. However, that is just my personal view; or are we going with a very simplistic and minimalist design for the library?

Reply:
The truth is, the reason why I don't want those things in the code is because they are heavily tailored to innopoints. There's actually a lot of conventions regarding cards, but this card barely satisfies the image one. So it's either we remove all those styles or rewrite the card properly. And I'm not up for the latter

Reply:
So since you added that `tight` prop, the rest of the contents of the Card are no longer necessary in my opinion

Reply:
I'm not sure what "properly" exactly means in this context, or what the conventions are, so I'll just go with the first option then.